viewer/static_src/css/crawler.less

```diff
@@ -0,0 +1,65 @@
+.o-radio_buttons {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+
+  div {
+    flex: 50%;
+  }
+
+  label {
+    font-weight: bold;
+
+    > span {
+      display: block;
+      margin-top: 3px;
+      margin-bottom: 15px;
+      font-weight: normal;
+    }
+  }
+}
+
+.results_list {
+  li {
+    list-style-type: none;
+  }
+
+  .results_item {
+    padding: 24px 0 !important;
+
+    + .results_item {
+      border-top: 1px solid var(--gray-40);
+    }
+  }
+}
+
+.u-truncate {
+  max-width: none !important;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  list-style-position: inside;
+  margin-bottom: 15px;
+}
+
+.u-crawl_text {
+  font-weight: bold;
+  padding-right: 5px;
+  margin-bottom: 15px;
+}
+
+.u-crawl_date {
+  display: flex;
+}
+
+.u-help_text {
+  margin-top: 15px;
+  margin-bottom: 30px;
+}
```

Comment:
Ideally, these should be updated to `u-crawl-text`, `u-crawl-date`, and `u-help-text`, unless they are elements, in which case they should be separated by a double underscore. E.g. if `u-crawl` is the block, `u-crawl__date` would be the element.

Reply:
Thanks @anselmbradford! I'm still getting used to the new naming conventions. I took a stab at this in eed3133: I made `crawl-date` an element (the div showing "Data last updated") and `crawl-date__text` its child text. The `help-text` element is distinct from these and is just a way to style the "Check out the Common searches page for examples". Which of these require a `u-` prefix -- is that only for utility mixins that are used throughout a set of pages?

Reply:
Essentially yea… since we divvied a lot up into atoms (`a-`), molecules (`m-`), and organisms (`o-`), `u-` is a way to have a fourth bucket for things that don't fit as a component and are more of a one-off utility.